src/Datadog.Trace/Configuration/GlobalSettings.cs

```diff
@@ -42,6 +49,21 @@ internal GlobalSettings(IConfigurationSource source)
         /// <seealso cref="ConfigurationKeys.DebugEnabled"/>
         public bool DebugEnabled { get; private set; }
 
+        /// <summary>
+        /// Gets the vendor plugin path.
+        /// </summary>
+        public string VendorPluginName { get; private set; }
```

Comment:
We may want to keep the PluginName more generic and remove the `Vendor` prefix, because there might be plugin use cases that are not vendor specific.

Reply:
+1 on this